### PR TITLE
Add script to collect camera trace

### PR DESCRIPTION
**Pull request by j-leungyy**
Add script to collect camera trace

### CODE REVIEW

**Comment by LPBourret** [comment]
i.e. 
//*********************************************************
//
// Copyright (c) Microsoft. All rights reserved.
// This code is licensed under the Microsoft Public License.
// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
//
//*********************************************************

File: Trace/Readme.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Quickstart
+
+To collect traces:
+- Double-click on Trace.cmd.
+- Run your scenario when told.
+- Press Enter and wait for the script to complete.
+- The folder containing the traces will pop up.
+
+Email traces:
+- Zip files (Right click > Send to > Compressed folder) and send.
```
Comment (saredd) [inline Trace/Readme.md]: Should this script auto zip the files?

**Comment by Sudhaushu** [comment]
I am not able to find out the current commit folder structure. Is this folder in the root path? Can we move it to a subfolder called "DiagnosticTools"

File: Trace/Readme.md (hunk 1 above)
Comment (Sudhaushu) [reply Trace/Readme.md]: can we add instructions on how to add a provider in this script?

Comment (j-leungyy) [reply Trace/Readme.md]: Saredd: will consider that as a improvement.
Sudhaushu: I don't thing we have much request from partner to add provider beside the one we have here so far.  Will consider if there are such request.

**Comment by j-leungyy** [comment]
Sudhaushu: All code here are under the directory "Trace"

File: Trace/Readme.md (hunk 1 above)
Comment (Sudhaushu) [reply Trace/Readme.md]: 360 camera DMFT partners keep asking how they can log from their DMFT.

**Comment by j-leungyy** [comment]
LPBourret: Do you know what's the guideline to add the copyright header?  script here https://github.com/Microsoft/busiotools doesn't have it.

**Pull request closed without merging**